ports/tbb/vcpkg.json

```
@@ -17,6 +17,12 @@
       "host": true
     }
   ],
+  "default-features": [
+    {
+      "name": "hwloc",
+      "platform": "!static & !osx"
+    }
```

```suggestion
    "name": "hwloc"
```
The following codes has `"supports": "!static & !osx",`.

---

... and that's why it must have `platform` here!